execution-plan/src/constants.rs

```diff
@@ -0,0 +1,26 @@
+use crate::Memory;
+use kittycad_execution_plan_traits::{Address, NumericPrimitive, Primitive};
+
+// Rust thinks this is dead-code but it will absolutely be used by consumers and
```

Comment:
It's inaccessible -- the consts themselves are `pub` but the module is private, so it is accurately marking this as dead code. If you make the `mod constants` into `pub mod constants` these symbols will be accessible, and the `dead_code` warning will go away.

Reply:
Ah ha